apps/dynamo/lib/dynamo/vnode_master.ex

```diff
@@ -45,6 +45,13 @@ defmodule Vnode.Master do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_cast({:repl_command, sender, msg}, state) do
```

Comment:
I don't think you need another command here. You can use ` def handle_cast({:command, sender, index, msg}, state) do` above.